nuxeo-drive-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-drive/nuxeo-drive-edit-button.html

```diff
@@ -66,7 +66,8 @@ <h1>[[i18n('driveEditButton.dialog.heading')]]</h1>
       },
 
       _isAvailable: function(doc, blob) {
-        return doc && blob && this.hasPermission(doc, 'Write');
+        return doc && blob && this.hasPermission(doc, 'Write')
+            && (blob.appLinks && Array.isArray(blob.appLinks) && blob.appLinks.length === 0);
```

Comment:
`Array.isArray(...)` could be enough to check if element exist?

Reply:
Yes, we can assume that appLinks is an array. If not, we'll got undefined which won't be equal to 0.